android/.gitignore

```diff
@@ -1,10 +0,0 @@
-*.iml
```

Comment:
Why remove .gitignore?

Reply:
I didn't mean to, or the .idea file I see is missing. I am adding them to my repo now and will commit changes